README.md

```diff
@@ -91,13 +184,108 @@ You then need to do a second call to the `signIn` mutation providing only the `o
 }
 ```
 
-## OAuth
+## OAuth / Login with Sorare
+
+If you want to make requests on behalf of other users you can rely on the [OAuth](https://oauth.net/2/) protocol.
+
+Please reach out to us first through our [Help Center](https://help.sorare.com/hc/en-us/requests/new) and ask us to generate you some OAuth credentials.
```

Comment:
to generate _you some OAuth credentials_.
-> to generate _some OAuth credentials for you_.

Reply:
In order to use our OAuth API, we will need to issue you a Client ID and Secret for your application. You can request one through our [Help Center](https://help.sorare.com/hc/en-us/requests/new) with the following information:
```
1. A unique name for your application
2. One or more callback URLs (e.g., https://yourapp.com/oauth/sorare/callback)
3. A logo for your application in PNG format
```